### ISSUE TITLE
@Environment(\.dismiss) is not working correctly

### ISSUE THREAD

**Issue by dm-katsubo** [CoolONEOfficial/NativePartialSheet#2]
`isPresented` state is not updated correctly when using `@Environment(\.dismiss)`

**Comment by CoolONEOfficial** [comment]
Could you provide some example of incorrect behavior? It would help me to fix it 😌

**Comment by CoolONEOfficial** [comment]
Here is minimal example of usage `@Environment(\.dismiss)`:
```
struct ContentView: View {
    @State var isPresented = false

    var body: some View {
        Text("Open sheet")
            .onTapGesture {
                isPresented = true
            }
            .sheet(isPresented: $isPresented) {
                TestView()
            }
            .presentationDetents([ .large, .medium ])
    }
}

struct TestView: View {
    @Environment(\.dismiss) var dismiss
    
    var body: some View {
        Button("test", action: {
            dismiss()})
    }
}
```

**Comment by CoolONEOfficial** [comment]
fixed in [2.0.3 version](https://github.com/CoolONEOfficial/NativePartialSheet/releases/tag/2.0.3)
